zebra-rpc/build.rs

```diff
@@ -4,12 +4,50 @@ use std::{env, fs, path::PathBuf, process::Command};
 const ZALLET_COMMIT: Option<&str> = None;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let out_dir = env::var("OUT_DIR").map(PathBuf::from);
-    tonic_prost_build::configure()
-        .type_attribute(".", "#[derive(serde::Deserialize, serde::Serialize)]")
-        .file_descriptor_set_path(out_dir.unwrap().join("indexer_descriptor.bin"))
-        .compile_protos(&["proto/indexer.proto"], &[""])?;
+    build_or_copy_proto()?;
+    build_zallet_for_qa_tests();
 
+    Ok(())
+}
+
+fn build_or_copy_proto() -> Result<(), Box<dyn std::error::Error>> {
+    let out_dir = env::var("OUT_DIR")
+        .map(PathBuf::from)
+        .expect("requires OUT_DIR environment variable definition");
+    let file_names = ["indexer_descriptor.bin", "zebra.indexer.rpc.rs"];
+
+    let is_protoc_available = env::var_os("PROTOC")
```

Comment:
We skip recompilation for downstream users by not attempting to rebuild if the proto files are absent: https://github.com/zcash/librustzcash/blob/796abb548fc8366738f16a5252fd026d5bd92652/zcash_client_memory/build.rs#L15

Reply:
Addressed in https://github.com/ZcashFoundation/zebra/pull/9819/commits/b820f2b87017b6035e6f7d45593a86846778943d.